rfc6962-bis.xml

```diff
@@ -846,46 +851,51 @@ messages.
         <t>
           POST https://&lt;log server&gt;/ct/v1/add-chain
         </t>
-
         <t>
-          Inputs:
-
-	  <list>
-	    <t><list style="hanging">
-	      <t hangText="chain:">
-		An array of base64-encoded certificates. The first element is the end-entity certificate; the second chains to the first and so on to the last, which is either the root certificate or a certificate that chains to a known root certificate.
-	      </t>
-	    </list></t>
-	  </list>
-        </t>
-
-        <t>
-          Outputs:
-
-          <list>
-	    <t><list style="hanging">
-	      <t hangText="sct_version:">
-		The version of the SignedCertificateTimestamp structure, in decimal. A compliant v1 implementation MUST NOT expect this to be 0 (i.e., v1).
-	      </t>
-	      <t hangText="id:">
-		The log ID, base64 encoded.
-	      </t>
-	      <t hangText="timestamp:">
-		The SCT timestamp, in decimal.
-	      </t>
-	      <t hangText="extensions:">
-		An opaque type for future expansion. It is likely that not all participants will need to understand data in this field. Logs should set this to the empty string. Clients should decode the base64-encoded data and include it in the SCT.
-	      </t>
-	      <t hangText="signature:">
-		The SCT signature, base64 encoded.
-	      </t>
-            </list></t>
+	  <list style="hanging">
+            <t hangText="Inputs:">
+	      <list style="hanging">
+		<t hangText="chain:">
+		  An array of base64-encoded certificates. The first element is the end-entity certificate; the second chains to the first and so on to the last, which is either the root certificate or a certificate that chains to a known root certificate.
+		</t>
+	      </list>
+	    </t>
+            <t hangText="Outputs:">
+	      <list style="hanging">
+		<t hangText="sct_version:">
+		  The version of the SignedCertificateTimestamp structure, in decimal. A compliant v1 implementation MUST NOT expect this to be 0 (i.e., v1).
+		</t>
+		<t hangText="id:">
+		  The log ID, base64 encoded.
+		</t>
+		<t hangText="timestamp:">
+		  The SCT timestamp, in decimal.
+		</t>
+		<t hangText="extensions:">
+		  An opaque type for future expansion. It is likely that not all participants will need to understand data in this field. Logs should set this to the empty string. Clients should decode the base64-encoded data and include it in the SCT.
+		</t>
+		<t hangText="signature:">
+		  The SCT signature, base64 encoded.
+		</t>
+              </list>
+	    </t>
+            <t hangText="Errors:">
```

Comment:
Ah, each would be present independently, I see.
In that case, unknown_root and bad_chain to match 'sct_version' above?